src/frontend/src/Components/routing/RouteSearch.js

```diff
@@ -84,7 +85,10 @@ const RouteSearch = forwardRef((props, ref) => {
 
     const shortestRoute = await getRoute(points, true);
     if (shortestRoute && shortestRoute.routeFound) {
-      routes.push(shortestRoute);
+      const isFastestIsShortest = compareRouteDistance(fastestRoute, shortestRoute);
+      if(! isFastestIsShortest){
+        routes.push(shortestRoute);
```

Comment:
Please use a better name for this const here

Reply:
is there an extra space before the new variable here?
![image](https://github.com/user-attachments/assets/be42f39a-1214-4e6f-b0c8-9ebcbb050256)

Reply:
Yes, I have removed it.